Project.toml

```diff
@@ -66,3 +66,12 @@ StatsBase = "0.32, 0.33, 0.34"
 StatsFuns = "0.8, 0.9, 1"
 Tracker = "0.2.3"
 julia = "1.7"
+
+[extensions]
+DynamicHMCExt = "DynamicHMC"
+
+[extras]
+DynamicHMC="bbc10e6e-7c05-544b-b16e-64fede858acb"
+
+[weakdeps]
+DynamicHMC="bbc10e6e-7c05-544b-b16e-64fede858acb"
```

Comment:
We need a compat entry now that it's a (weak) dependency (which IMO is good since it allows us to rule out incompatible DynamicHMC versions in a much better way) 🙂

Reply:
done!